packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -338,6 +340,17 @@ QUnit.module('Chat', moduleConfig, () => {
             assert.strictEqual($(lastBubble).text(), text);
         });
     });
+
+    QUnit.module('Methods', () => {
+        QUnit.test('The textarea input element must be active after the focus() method is invoked', function(assert) {
+            this.instance.focus();
+
+            const root = document.querySelector('#qunit-fixture');
+            const activeElement = root.shadowRoot ? root.shadowRoot.activeElement : document.activeElement;
```

Comment:
u can use `domAdapter.getActiveElement()`

Reply:
I saw that $(‘#qunit-fixture’).get(0) has different values depending on the mode (shadow or not), it's hard to understand without a debugger, so I'll leave the code as it is as I find it more intuitive